internal/tui/components/app/app.go

```diff
@@ -211,8 +211,9 @@ func (m Model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 				}
 			case nomad.ExecPage:
 				m.getCurrentPageModel().SetInputPrefix("Enter command: ")
-			case nomad.AllocAdminConfirmPage:
-				// always make user go down one to confirm
+			case nomad.AllocAdminConfirmPage, nomad.JobAdminConfirmPage, nomad.JobAdminPage,  nomad.AllocAdminPage:
+				// always start at the top of the list for admin pages
+				// avoids surprises, consistently focusing on the first item
```

Comment:
I don't think we want this for the job admin page or alloc admin page - if someone is performing the same admin task across many entities, they'll want the focus on the same one each time similar to other pages

Related: https://github.com/robinovitch61/wander/issues/132